### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`version++`
`polymer++`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -22,15 +22,15 @@
   "devDependencies": {
     "webcomponentsjs": "webcomponents/webcomponentsjs#~0.6.1",
     "web-component-tester": "Polymer/web-component-tester#^2.2.3",
```
Comment (cdata) [inline bower.json]: This should be `"*"`

File: bower.json [hunk 2]
```diff
@@ -22,15 +22,15 @@
   "devDependencies": {
     "webcomponentsjs": "webcomponents/webcomponentsjs#~0.6.1",
```
Comment (cdata) [inline bower.json]: This should be `^0.6.0`

File: demo/index.html [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
     <link rel="stylesheet" href="../../paper-styles/demo.css">
 
-    <style is="x-style">
+    <style is="custom-style">
       paper-checkbox.blue {
         --paper-checkbox-unchecked-color: var(--dark-primary-color);
         --paper-checkbox-unchecked-ink-color: var(--dark-primary-color);
```
Comment (cdata) [inline demo/index.html]: These should probably be inline defaults below.

File: paper-checkbox.html [hunk 4]
```diff
@@ -45,7 +45,7 @@
 -->
 
 <dom-module id="paper-checkbox">
-  <style is="x-style">
+  <style is="custom-style">
     * {
       --paper-checkbox-unchecked-color: var(--primary-text-color);
       --paper-checkbox-unchecked-ink-color: var(--primary-text-color);
```
Comment (cdata) [inline paper-checkbox.html]: These should probably be inline defaults below.

**Comment by notwaldorf** [comment]
Done! (punting the default values for now)

**Comment by cdata** [comment]
Derp.

**Comment by cdata** [comment]
:+1:

**Pull request merged**